tests/test_qpu_qiskit.py

```diff
@@ -21,17 +24,140 @@
 )
 from openqaoa.utilities import X_mixer_hamiltonian
 from openqaoa.problems import NumberPartition
+from openqaoa import QAOA
+
+
+class TestingQAOAQiskitQPUBackendAzure(unittest.TestCase):
+    
+    """This Object tests the QAOA Qiskit QPU Backend object with the Azure 
+    Device object. This checks that the use of qiskit to send circuits to Azure
+    is working as intended.
+    
+    The Azure CLI has to be configured beforehand to run these tests.
+    """
+    
+    @pytest.mark.qpu
+    def setUp(self):
+        
+        bashCommand = "az resource list"
+        process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
+        output, error = process.communicate()
+        
+        if error is not None:
+            print(error)
+            raise Exception('You must have the Azure CLI installed and must be logged in to use the Azure Quantum Backends')
+        else:
+            output_json = json.loads(output)
```

Comment:
@shahidee44 have you already created a resource for this?

Reply:
No. Currently we're just using the TestingOpenQAOA Workspace for tests.